samples/FunctionApp/FunctionApp.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.21.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.13.1" />
```

Comment:
Update this to 1.14.0? - https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk/